docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -1110,6 +1111,34 @@ This will override the default directives as specified in the ``compiler_directi
 Note that explicit per-file or local directives as explained above take precedence over the
 values passed to ``cythonize``.
 
+.. _cline_in_traceback:
+
+C line numbers in tracebacks
+============================
+
+To provide more detailed debug information, Python tracebacks of Cython modules
+show the C line where the exception originated (or was propagated). This feature is not
+entirely for free and can visibly increase the C compile time as well as adding 0-5% to the
+size of the binary extension module. It can therefore be disabled using a C macro.
+
+* ``CYTHON_CLINE_IN_TRACEBACK=1`` always shows the C line number in tracebacks,
+* ``CYTHON_CLINE_IN_TRACEBACK=0`` never shows the C line number in tracebacks,
+
+If the macro is *not* defined by the build setup or ``CFLAGS``, the feature is included and
+can be enabled and disabled at runtime, at the before mentioned cost of longer C compile
+times and larger extension modules. This is the default behaviour.
```

Comment:
I am sorry for being pain. If I am not wrong (probably I am) I cannot replicate it. The documentation is saying that if macro is not defined by build setup or CFLAGS I should be able to enable it at "runtime". Tried via `cython_runtime` and failed:

delete.pyx
```cython
import cython_runtime

cython_runtime.c_line_in_traceback=True

cdef bleh():
    raise ValueError()

bleh()
```

setup.py
```python
from setuptools import setup, Extension
from Cython.Build import cythonize
import numpy

from Cython.Compiler import Options

extensions = [
    Extension("*", ["delete.pyx"],
    ),
]

setup(
    name="My hello app",
    ext_modules=cythonize(extensions)
)
```

```
python -c 'import delete'

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "delete.pyx", line 8, in init delete
    bleh()
  File "delete.pyx", line 6, in delete.bleh
    raise ValueError()
ValueError
```

Reply:
I also tried to enable it in `setup.py` with the same result:
```python
from setuptools import setup, Extension
from Cython.Build import cythonize
import numpy

from Cython.Compiler import Options


extensions = [
    Extension("*", ["delete.pyx"],
    ),
]

setup(
    name="My hello app",
    ext_modules=cythonize(extensions, c_line_in_traceback=True),
)
```

Reply:
> ```cython
> cython_runtime.c_line_in_traceback=True
> ```
This should read
> ```cython
> cython_runtime.cline_in_traceback=True
> ```

Reply:
> > ```cython
> > cython_runtime.c_line_in_traceback=True
> > ```
> 
> This should read
> 
> > ```cython
> > cython_runtime.cline_in_traceback=True
> > ```

Yes. I thought that it has identical name as the option but not... Now it works.